sdk/ml/azure-ai-ml/azure/ai/ml/_ml_client.py

```diff
@@ -374,8 +385,14 @@ def __init__(
         self._models = ModelOperations(
             self._operation_scope,
             self._operation_config,
-            self._service_client_10_2021_dataplanepreview if registry_name else self._service_client_05_2022,
+            self._service_client_10_2021_dataplanepreview
+            if registry_name
+            else self._service_client_02_2023_preview
```

Comment:
Does this need to be `service_client_04_2023_preview` instead?

Reply:
I have the same comment, but this thread is resolved. What is the outcome? I would have thought it would use the new API version we've taken a dependency on.

Reply:
I thought I reposnded to it,may be I forgot to save it. There is no difference between 02_2023 and 04_2023 at this point of time for model package and this is only used for private feature flag, this will be revisited when asset stages are ready and we can use 04 then. With 04 not much testing happen yet.